src/Visitor/Php/Symfony/FormTypePlaceholder.php

```diff
@@ -51,6 +51,12 @@ public function enterNode(Node $node)
                 $placeholderNode = $item;
             } elseif ('attr' === $item->key->value && $item->value instanceof Node\Expr\Array_) {
                 foreach ($item->value->items as $attrValue) {
+                    if ($this->isNotSupported($attrValue)) {
+                        $this->addError($node, 'Form attribute key is a variable.');
```

Comment:
You cannot write this error message here. You technically dont know *why* it is not supported. 

What about reverting `FormTrait` and add

```php
if ($attrValue->key instanceof Node\Expr\Variable) {
  $this->addError($node, 'Form attribute key is a variable.');
}
```